tests/test_history.py

```diff
@@ -1,96 +1,103 @@
 """Tests the xonsh history."""
+# pylint: disable=protected-access
 from __future__ import unicode_literals, print_function
 import io
 import os
 import sys
 
 import nose
-from nose.tools import assert_equal, assert_true
+from nose import tools as nose_tools
```

Comment:
I am not sure that I am a fan of this, since it is different that the way I have seen it done in most other projects (numpy, scipy, etc).  What is the reason for renaming the module?